usagov/usagov-2021
# USAGOV-150: USAGOV-150 - Increase height of the USWDS Components dialog

Bigger opening / window for the "Select USWDS Components to Embed"


https://cm-jira.usa.gov/browse/USAGOV-150

## Description
Increased the height of the pop-up modal dialog-window for USWDS Components in the WYSIWYG 

## Type of Changes


## Testing Instructions
Merge branch, flush cache. Then go to add a new Basic Page. Click the "USWDS Components" button in the WYSIWYG toolbar. You should now see the modal pop-up window is larger compared to Stage/Prod.

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
